package.json

```diff
@@ -0,0 +1,15 @@
+﻿{
+	"name": "space.exodrifter.unity-rumor",
+	"version": "5.0.0",
+	"description": "Rumor is a free, open source narrative content framework meant to be integrated into any Unity3D game that needs a dialog system. It is easy to diff, easy to read, and it is very flexible!",
+	"displayName": "Unity Rumor",
+	"unity": "2020.3",
+	"author": {
+		"name": "exodrifter",
+		"email": "exodrifter@gmail.com",
```

Comment:
I don't use this email address anymore, you should change it to `code@exodrifter.space`

Reply:
Yeah, sorry, I looked into some older projects of yours and through itch and your website to find those information :D